app/views/aid_applications/duplicates/_data_row.erb

```diff
@@ -0,0 +1,4 @@
+<tr id="<%= dom_id(aid_application) + rand(99).to_s %>" >
```

Comment:
What is the purpose of the `id` element?

Reply:
This is to ensure that the ids are unique for iteration